docs/yamls/alias.md

```diff
@@ -0,0 +1,29 @@
+## Alias 
+
+### Definitions
+
+| Field             | Meaning                                                                                                                                                                                                      |
+| ----------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| apiVersion        | Specify what version of the API to use, for example `"v1"`                                                                                                                                                   |
+| kind              | Specifies which structure the file represents, in this case it would be `alias`                                                                                                                              |
+| meta              | Metadata of Type                                                                                                                                                                                             |
+| &rarr;name        | Type Name of the Alias                                                                                                                                                                                       |
```

Comment:
"Type Name of the Alias"?  
Also, you should point out that the name of the alias is something like "childDApp.childDAppBoundary"